gtk/po/af.po

```diff
@@ -381,9 +381,9 @@ msgid ""
 "By default the longest title is chosen.\n"
 "This is often the feature title of a DVD."
 msgstr ""
-"Stel die titel vir enkodering.\n"
-"Die langste titel word by verstek gekies.\n"
-"Dit is gewoonlik die hooftitel van ’n DVD."
+"Stel die title vir enkodering.\n"
```

Comment:
I didn't mean to include this change -- the reason the file is changed at all is because of the earlier hunk. I'm going to just go through and drop the gtk/po/ files and trust that the localization system has a away to handle them.